src/System.Private.CoreLib/shared/System/String.Manipulation.cs

```diff
@@ -34,14 +34,7 @@ private static unsafe void FillStringChecked(string dest, int destPos, string sr
             }
         }
 
-        public static string Concat(object? arg0)
-        {
-            if (arg0 == null)
-            {
-                return string.Empty;
-            }
-            return arg0.ToString();
-        }
+        public static string Concat(object? arg0) => arg0?.ToString() ?? string.Empty;
```

Comment:
Make sure to add test coverage for this behavior change as well - if we do not have it already.

Reply:
with an object that returns null from ToString()? Ok, sounds good.

Reply:
We don't have that coverage, we only pass down null to the overrides that take more than 1 arg. I'll submit a PR that waits until this guy makes it to corefx.